### DISCUSSION TITLE
How do you set a list/array as a value in JSON?

### DISCUSSION THREAD

**Discussion by Jonseed** [lucafoscili/comfyui-lf#92] [Q&A]
I have a list in string format like this, `['item1','item2','item3']`, but when I set it to a value your create/set JSON node, I get a value that is a single string like this: `"['item1','item2','item3']"`. If I convert it to a true list first, your create/set JSON node doesn't accept it as a value input.

**Comment by lucafoscili** [comment]
Hmm I'll have to check tomorrow to be sure but I think I handled lists there, so if the result is a string there has to be a bug.
I haven't tested that node thoroughly as it's not in one of my workflows currently.
I'll take a look and let you know!

**Comment by lucafoscili** [comment]
I'm testing out a fix - what do you think?
WriteJSON nodes didn't have "key", it was added after the workflow finished the execution (I didn't know that it was the same memory reference, pretty cool!).

![Screenshot 2024-10-15 090751](https://github.com/user-attachments/assets/8fce5b65-a960-4c73-9f4a-4dcbabf521b4)

**Reply by Jonseed** [reply]
That looks like it could work.

**Reply by lucafoscili** [reply]
Great, I'll push a new release this evening with this fix!
